src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -95,34 +95,47 @@
 
     <!-- file upload components -->
     <v-row
-      v-for="(num, index) in numUploads"
-      :key="authorization.files[index]?.fileKey"
       class="upload-file-row mt-4 mb-n2"
       no-gutters
     >
       <v-col
         cols="12"
         sm="3"
       >
-        <label v-if="index === 0">Upload File</label>
+        <label>Upload File</label>
       </v-col>
 
       <v-col
         cols="12"
         sm="9"
       >
         <FileUploadPreview
+          v-for="(num, index) in numUploads"
+          :key="authorization.files[index]?.fileKey"
           inputFileLabel="Continuation authorization"
           :maxSize="MAX_FILE_SIZE"
           :pdfPageSize="PdfPageSize.LETTER_SIZE"
-          :hint="authorization.files[index]?.file ? 'File uploaded' : undefined"
+          :label="authorization.files[index]?.file ? 'File uploaded' : undefined"
+          hint=" "
           :inputFile="authorization.files[index]?.file"
           :showErrors="getShowErrors"
           :customErrorMessage.sync="customErrorMessage[index]"
           :isRequired="getShowErrors && (index === 0)"
           @fileValidity="onFileValidity($event)"
           @fileSelected="onFileSelected(index, $event)"
         />
+
+        <v-btn
+          id="add-another-file-button"
+          outlined
+          color="primary"
+          class="btn-outlined-primary mt-6 ml-8"
+          :disabled="false"
+          @click="void 0"
```

Comment:
Interesting syntax here. I've never seen this. What does this do exactly?

Reply:
It does exactly nothing 😆 

I didn't want to delete this prop so I gave a tempory "nothing" action. This is something I'm still working on.

Reply:
🤣 🤣 All right, sounds good!

Reply:
Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void#:~:text=using%20void(0)%20(which%20is%20equivalent%20to%20void%200)

Reply:
this is interesting! :)